armi/bookkeeping/db/database3.py

```diff
@@ -100,6 +94,168 @@ def getH5GroupName(cycle: int, timeNode: int, statePointName: str = None) -> str
     return "c{:0>2}n{:0>2}{}".format(cycle, timeNode, statePointName or "")
 
 
+class JaggedArray:
+    def __init__(self, jaggedData, paramName):
+        """
+        Take a list of numpy arrays or lists and flatten them into a single 1D array.
+
```

Comment:
It is more _typical_ to have a long docstring on the class level, not the `__init__` method. Can we move this docstring up, so the class has a docstring?